order-service/src/test/java/com/minhaz/orders101/endpoints/OrdersEndpointIntegrationTest.java

```diff
@@ -154,6 +154,14 @@ public void testDELETERequest() {
     assertThat(response.getStatusCode()).isEqualTo(HttpStatus.OK);
   }
 
+  @Test
+  @Disabled("can not reach end point")
```

Comment:
what's happening in this MR? 
(doesn't seem to relate to PATCH stock availability)